generator/src/compile-elm.js

```
@@ -133,6 +133,12 @@ async function compileElm(options, elmEntrypointPath, outputPath, cwd) {
 
 function spawnElmMake(options, elmEntrypointPath, outputPath, cwd) {
   return new Promise(async (resolve, reject) => {
+    try {
+      await fsPromises.unlink(outputPath);
+    } catch (e) {
+      /* File may not exist, so ignore errors */
+    }
```

No real need to check whether the file already exists, first, IMO. "File exists" is an ephemeral state anyway -- the answer could change at any moment, so I changed this to just try to delete it.

I removed the options argument because there is no options argument to unlink(), at least not in the versions of Node I looked up.

https://nodejs.org/docs/latest-v20.x/api/fs.html#fsunlinkpath-callback

---

Agree on the check being redundant if we ignore the error anyway